### PR TITLE
CHAT-733 : select the room after its creation

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
This PR render the newly created room directly (instead of waiting for the websockets message) and select it.

### CODE REVIEW

File: application/src/main/webapp/js/chat.js [hunk 1]
```diff
@@ -1242,7 +1242,14 @@ var chatApplication = new ChatApplication();
 
       var users = chatApplication.username + ',' + $('#team-add-user').val();
 
-      chatApplication.saveTeamRoom(teamName, teamId, users);
+      chatApplication.saveTeamRoom(teamName, teamId, users, function(data) {
+        // if it is a new room...
+        if(teamId === "---") {
+          // display it in the list of rooms and select it
+          chatApplication.renderRooms();
```
Comment (trongtt) [inline application/src/main/webapp/js/chat.js]: we don't need to do renderRooms() as it has been invoked when new room is added.

Comment (thomasdelhomenie) [reply application/src/main/webapp/js/chat.js]: I don't think it is not invoked directly after the creation, but when the websocket message is received, which could lead to a delay between the creation and the display. That's why I render the rooms directly on client side. The room is displayed directly and I am sure I can select it.

Comment (trongtt) [reply application/src/main/webapp/js/chat.js]: The point is that the new room data is added at the time of receiving websocket message, even for creator. So there might be an issue with the delay, and this renderRooms() doesn't help.

Comment (thomasdelhomenie) [reply application/src/main/webapp/js/chat.js]: After discussions and more testing, the room will always be rendered when the callback is called (by the websocket listener), so the renderRooms call is useless. Thanks @trongtt

**Pull request merged**